src/pages/[platform]/build-a-backend/add-aws-services/logging/set-up-logging/index.mdx

```diff
@@ -104,29 +104,20 @@ You will need to create a log group in Amazon CloudWatch to send logs to. You ca
 
 Below is a sample CDK construct to create the Amazon CloudWatch log group as well as creating and assigning the permission policies to Amplify roles.
 
-The `<log-group-name>` and `<region>` configured in the CDK construct will be used later to initialize the Amplify Logger plugin.
+First create the file `amplify/custom/RemoteLoggingConstraintsConstruct.resource.ts` with the following:
```

Comment:
Is there a typo or is the path intended to be `RemoteLoggingConstraintsConstruct.resource.ts`

Reply:
nice catch, it was a typo for the folder/file structure `amplify/custom/RemoteLoggingConstraintsConstruct/resource.ts`. fixed the typo and updated the wording to match the custom CDK examples under https://docs.amplify.aws/swift/build-a-backend/add-aws-services/custom-resources/